app/js/portal/config.js

```diff
@@ -32,6 +32,12 @@ function Config(cfpLoadingBarProvider, InsightsConfigProvider) {
     InsightsConfigProvider.setPortal(true);
     InsightsConfigProvider.setGettingStartedLink(
         'https://access.redhat.com/products/red-hat-insights#getstarted');
+
+    /* Used for Satellite integration
+    InsightsConfigProvider.setAnsibleRunner(function ($location, planId, customize) {
```

Comment:
@lphiri this is the hook for Satellite to provide which determines the URL of e.g. the job execution form

Reply:
Would this be a redirect or an API call?

Reply:
I am fine either way. Depends on what @lphiri prefers. Let me know if you need any other dependencies passed into the callback.